src/signals/incident-management/containers/IncidentCategoryContainer/components/Add/index.js

```diff
@@ -38,10 +38,13 @@ class Add extends React.Component { // eslint-disable-line react/prefer-stateles
                 <div>
                   <FieldControlWrapper
                     render={SelectInput}
-                    name="sub"
+                    name="sub_category"
                     display="Subcategorie"
-                    control={this.categoryForm.get('sub')}
-                    values={subcategoryList}
+                    control={this.categoryForm.get('sub_category')}
+                    values={[{
```

Comment:
Not really the best place to add the default entry, rather place add it in the reducer or container

Reply:
ja klopt ik heb het hele ding gerefactored: nu nog unit testen maken